### PR TITLE
bump fedora containerdisk to 36

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
bump fedora containerdisk to 36

**Release note**:
```
bump fedora containerdisk to 36
```

### CODE REVIEW

File: templates/fedora.tpl.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ metadata:
     template.kubevirt.io/version: v1alpha1
     defaults.template.kubevirt.io/disk: rootdisk
     template.kubevirt.io/containerdisks: |
-      quay.io/containerdisks/fedora:35
+      quay.io/containerdisks/fedora:36
```
Comment (ksimon1) [inline templates/fedora.tpl.yaml:19]: @dominikholler is it ok to keep only 36, or do we want to keep both?

Comment (dominikholler) [reply templates/fedora.tpl.yaml:19]: Let's go with only 36 for  now, to be sure the version 36 is shown in UI.

@upalatucci if there would be multiple containerdisks, which one would be shown in the UI?

**Review by dominikholler — APPROVED** [review]
/lgtm
fredora 36 reached in https://quay.io/repository/containerdisks/fedora?tab=tags

**Pull request merged**